packages/react-sdk/src/components/index.ts

```diff
@@ -45,7 +45,16 @@ let registeredComponents: RegisteredComponents | null = null;
 
 export type ComponentName = keyof typeof components;
 
-export const componentNames = Object.keys(components) as ComponentName[];
+const componentNames = Object.keys(components) as ComponentName[];
+
+export const getComponentNames = (): ComponentName[] => {
```

Comment:
I think we should use "use" instead of "get" as we assume that they are called inside React components
after init cc @kof